src/screens/SettingsSheet/components/DevSection.tsx

```diff
@@ -181,13 +180,9 @@ const DevSection = () => {
   }, [walletNotificationSettings]);
 
   const clearPendingTransactions = async () => {
-    // clear local storage
-    saveLocalPendingTransactions([], accountAddress, Network.mainnet);
-    // clear redux
-    dispatch({
-      payload: [],
-      type: DATA_UPDATE_PENDING_TRANSACTIONS_SUCCESS,
-    });
+    // clear pending txs
+
+    //  TODO clear pending transactions
 
     // reset nonces
     const { clearNonces } = nonceStore.getState();
```

Comment:
this clears all nonces ever which is kinda fine since we recalc but kinda no ideal 

🤔